src/models/tools/table/table-content.ts

```diff
@@ -440,7 +441,7 @@ export const TableContentModel = types
             links
       });
     },
-    removeGeometryLinks(geometryIds: string | string[], links?: ILinkProperties) {
+    removeGeometryLink(geometryIds: string | string[], links?: ILinkProperties) {
```

Comment:
```suggestion
    removeGeometryLink(geometryId: string, links?: ILinkProperties) {
```
Let's not imply that removing multiple links is supported.